CHANGELOG.md

```diff
@@ -1,219 +1,274 @@
+- Supported tag on the forge.
 
-### Features
+### Changed
 
-- `chocolateysource` - explicitly require location in ensure ([MODULES-3430](https://tickets.puppet.com/browse/MODULES-3430))
 - set ignore package exit codes when Chocolatey is on version 0.9.10+ ([MODULES-3880](https://tickets.puppet.com/browse/MODULES-3880))
 
-### Bug Fixes
+### Fixed
 
-- Fix: Ensure config file exists before `chocolateyfeature`, `chocolateyconfig`, or `chocolateysource` ([MODULES-3677](https://tickets.puppet.com/browse/MODULES-3677))
-- Fix: `chocolateysource` - ensure flush when disabling source ([MODULES-3430](https://tickets.puppet.com/browse/MODULES-3430))
-- Fix: `chocolateysource` - erroneous user sync messages ([MODULES-3758](https://tickets.puppet.com/browse/MODULES-3758))
+- Ensure config file exists before `chocolateyfeature`, `chocolateyconfig`, or `chocolateysource` ([MODULES-3677](https://tickets.puppet.com/browse/MODULES-3677))
+- `chocolateysource` - ensure flush when disabling source ([MODULES-3430](https://tickets.puppet.com/browse/MODULES-3430))
+- `chocolateysource` - erroneous user sync messages ([MODULES-3758](https://tickets.puppet.com/browse/MODULES-3758))
 
+## [0.8.0] - Unsupported 2016-07-13
 
-## 2016-07-13 Unsupported Release 0.8.0
-
-This brings the unsupported puppetlabs-chocolatey provider on par with the approved chocolatey-chocolatey at 1.2.6 and adds additional features.
+### Added
 
 - Includes community module releases up to 1.2.6 (changelog below).
 - Manage features - `chocolateyfeature` - see [MODULES-3034](https://tickets.puppet.com/browse/MODULES-3034)
 - Manage config settings - `chocolateyconfig` - see [MODULES-3035](https://tickets.puppet.com/browse/MODULES-3035)
 
+## [0.7.0] - Unsupported 2016-06-01
 
-## 2016-06-01 Unsupported Release 0.7.0
+### Added
 
 - Manage sources - `chocolateysource` - see [MODULES-3037](https://tickets.puppetlabs.com/browse/MODULES-3037)
-- Includes community module releases up to 1.2.1 (changelog below up to 1.2.1), plus these additional fixes:
-  - $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
-  - Fix: puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
+- Includes community module releases up to 1.2.1 (changelog below up to 1.2.1)
+
+### Fixed
 
+- $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
+- puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
 
-# Approved Community Module Changelog - Chocolatey Team
+# Note:
+The changelog below is for changes to the Community chocolatey-chocolatey module from which this current module takes over. It is retained below for historical purposes since for a couple releases this module did track its changes and bug fixes.
```

Comment:
This sentence doesn't read very well. I've reworded it below (which is how I understood it, please adjust if it's not quite right):

"The puppetlabs-chocolatey module replaces the community chocolatey-chocolatey module. We have retained its changelog below as there were a couple of releases where we tracked the new modules changes and bug fixes here"

* Community - lower case.